include/nbl/asset/filters/CRegionBlockFunctorFilter.h

```diff
@@ -59,6 +59,11 @@ class CRegionBlockFunctorFilter : public CImageFilter<CRegionBlockFunctorFilter<
 
 			CBasicImageFilterCommon::executePerBlock<ExecutionPolicy,Functor>(std::forward<ExecutionPolicy>(policy),state->image,*state->regionIterator,state->functor);
 
+			if constexpr(!ConstImage)
+			{
+				state->image->setContentHash(core::INVALID_HASH);
+			}
+
```

Comment:
does the FlattenRegions filter implement stuff in terms of `execute` defined here ?

Reply:
FlattenRegions don't use RegionBlockFunctor

Reply:
ok no problem then